source/renderer/app/components/voting/VotingNoWallets.tsx

```diff
@@ -41,13 +41,17 @@ export default class VotingNoWallets extends Component<Props> {
 
   render() {
     const { intl } = this.context;
-    const { onGoToCreateWalletClick, minVotingFunds } = this.props;
+    const {
+      onGoToCreateWalletClick,
+      minVotingFunds,
+      nextFundNumber,
+    } = this.props;
     return (
       <div className={styles.component}>
         <SVGInline svg={icon} className={styles.icon} />
         <h1>
           {intl.formatMessage(messages.headLine, {
-            nextVotingFundNumber: NEXT_VOTING_FUND_NUMBER,
+            nextVotingFundNumber: nextFundNumber,
```

Comment:
Potentially we can end up with the word "undefined" contained in the translated string since the `nextFundNumber` property is optional. We could tackle this by making this field required and somewhere above ensure it is present. We can also skip the whole `h1` message, but I guess the first suggestion is more appropriate.

This comment applies also to the rest of components having the optional `nextVotingFundNumber`

Reply:
I'll update to make it required. If `undefined` the translation will just display "Fund".